node/node_genesis.go

```diff
@@ -92,5 +94,6 @@ func AddNodeAddressesToGenesisAlloc(genesisAlloc core.GenesisAlloc) {
 		testBankFunds = testBankFunds.Mul(testBankFunds, big.NewInt(params.Ether))
 		address := common.HexToAddress(account.Address)
 		genesisAlloc[address] = core.GenesisAccount{Balance: testBankFunds}
+		utils.GetLogInstance().Info("****free money****", "address", address, "balance", testBankFunds)
```

Comment:
trying to fund free money to these address for the demo. followings are just logs of the expectation we are looking at


INFO [03-15|22:28:04.574] ****free money****                       port=9000 ip=127.0.0.1 address=0x1a3e7a44ee21101d7D64FBf29B0F6F1fc295F723 balance=1000000000000000000000
INFO [03-15|22:28:04.575] ****free money****                       port=9000 ip=127.0.0.1 address=0x10A02A0a6e95a676AE23e2db04BEa3D1B8b7ca2E balance=1000000000000000000000
INFO [03-15|22:28:04.575] ****free money****                       port=9000 ip=127.0.0.1 address=0x3e881F6C36A3A14a2D1816b0A5471d1caBB16F33 balance=1000000000000000000000
INFO [03-15|22:28:04.575] ****free money****                       port=9000 ip=127.0.0.1 address=0x9d72989b68777a1f3FfD6F1DB079f1928373eE52 balance=1000000000000000000000
INFO [03-15|22:28:04.575] ****free money****                       port=9000 ip=127.0.0.1 address=0x67957240b6eB045E17B47dcE98102f09aaC03435 balance=1000000000000000000000
INFO [03-15|22:28:04.575] ****free money****                       port=9000 ip=127.0.0.1 address=0xf70fBDB1AD002baDF19024785b1a4bf6F841F558 balance=1000000000000000000000
INFO [03-15|22:28:04.575] ****free money****                       port=9000 ip=127.0.0.1 address=0x3f1A559be93C9456Ca75712535Fd522f5EC22c6B balance=1000000000000000000000
INFO [03-15|22:28:04.575] ****free money****                       port=9000 ip=127.0.0.1 address=0xedD257B4e0F5e7d632c737f4277e93b64DC268FC balance=1000000000000000000000
INFO [03-15|22:28:04.575] ****free money****                       port=9000 ip=127.0.0.1 address=0x66A74477FC1dd0F4924ed943C1d2F1Dece3Ab138 balance=1000000000000000000000
INFO [03-15|22:28:04.575] ****free money****                       port=9000 ip=127.0.0.1 address=0x04178CdbCe3a9Ff9Ea385777aFc4b78B3E745281 balance=1000000000000000000000